dex/plan.sh

```diff
@@ -15,36 +16,15 @@ pkg_deps=(core/glibc)
 pkg_build_deps=(core/go core/git core/gcc)
 pkg_bin_dirs=(bin)
 
-do_before() {
-  GOPATH=$HAB_CACHE_SRC_PATH/$pkg_dirname
-  export GOPATH
-}
-
-do_prepare() {
-  export GO_LDFLAGS="-w -X $gopkg/version.Version=v$pkg_version"
-}
-
-do_download() {
-  return 0
-}
-
-do_verify() {
-  return 0
-}
-
-# Use unpack instead of download, so that plan-build can manage the
-# source path. This ensures us a clean checkout every time we build.
-do_unpack() {
-  git clone "$pkg_source" "$GOPATH/src/$gopkg"
-  ( cd "$GOPATH/src/$gopkg" || exit
-    git reset --hard "v$pkg_version"
-  )
-}
-
 do_build() {
-  go build --ldflags "${GO_LDFLAGS}" -o "$pkg_prefix/bin/dex" "$gopkg/cmd/dex"
+  cat >scripts/git-version <<EOF
+#/bin/sh
```

Comment:
Should this be `#!`? My guess is it works regardless, but probably best for clarity.

Reply:
`#!`?! :mouth  of course. Thanks.